### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by YoelAharoni12**
Finish semantic exrc

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/package-lock.json
```
[the remark this answers is not in the document]
Comment (rongoldinfeld) [reply .gitignore:1]: yes he sholud be in gitignore
and when we install a new library with npm, it Updated automatically.

**Review by rongoldinfeld — CHANGES_REQUESTED** [review]

File: src/style/index.less [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  margin-bottom: 3% !important;
+}
+
+.account-box {
+  margin: 0 !important;
+}
+.border-item {
+  border: thin solid transparent;
+
+  &:hover {
```
[the remark this answers is not in the document]
Comment (YoelAharoni12) [reply src/style/index.less:32]: represents the current selector parent, it also give u an options to do nesting with your element

File: src/scripts/Jquey.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+console.log($('.ui .menu'))
```
Comment (rongoldinfeld) [inline src/scripts/Jquey.js]: remove console.log

File: src/style/index.less [hunk 4]
```diff
@@ -0,0 +1,46 @@
+@padding: 0 !important;
```
Comment (rongoldinfeld) [inline src/style/index.less:1]: remove important from variable declaration

**Review by rongoldinfeld — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (rongoldinfeld) [reply .gitignore:1]: package-lock should be part of the project: 
1. provides security 
2. verifies which versions you actually installed

File: .gitignore [hunk 5]
```diff
@@ -1,5 +1,8 @@
 /package-lock.json
-/semantic/
+
 /node_modules/
 .idea
 /temp.html
+/semantic/dist/
```
Comment (rongoldinfeld) [inline .gitignore:6]: keep semantic css files but remove css,.map files from your src

File: src/style/index.less [hunk 6]
```diff
@@ -23,6 +24,9 @@
 .account-box {
   margin: 0 !important;
 }
+.orange-color{
+  color: #FF4500 !important;
```
Comment (rongoldinfeld) [inline src/style/index.less:28]: need the important?

File: src/style/index.less [hunk 7]
```diff
@@ -44,3 +48,6 @@
 .ui .card:hover .card-image {
   filter: opacity(80%);
 }
+.shekel-label::after{
+  content: "\20AA" !important;
```
Comment (rongoldinfeld) [inline src/style/index.less:52]: need the important?

File: src/amaRomach/index.html [hunk 8]
```diff
@@ -25,7 +25,7 @@
                 <div class="eight wide column search-bar">
                     <div class="ui fluid icon input">
                         <input type="text" placeholder="Search a very wide input...">
-                        <button class="ui icon primary button">
+                        <button class="ui icon romach secondary button">
```
Comment (rongoldinfeld) [inline src/amaRomach/index.html:28]: inside src folder create two folders:
1. home includes: home.html, home.css
2. card includes: cart.html, cart.less

File: src/style/index.less [hunk 9]
```diff
@@ -44,3 +48,6 @@
 .ui .card:hover .card-image {
   filter: opacity(80%);
 }
+.shekel-label::after{
```
Comment (rongoldinfeld) [inline src/style/index.less:51]: what does 
@siteFolder  : 'site';
@site       : 'amazon';

mean in theme.config?

Comment (rongoldinfeld) [reply src/style/index.less:51]: why do u need the .min.css files?

**Review by rongoldinfeld — CHANGES_REQUESTED** [review]

File: src/amaRomach/index.html [hunk 10]
```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (rongoldinfeld) [reply src/amaRomach/index.html:1]: DONE!

**Pull request merged**